### PR TITLE
Support ctrl-c for cancelling a command

### LINKED ISSUE #8: Support ctrl-c for cancelling a command

### PR DESCRIPTION
**Pull request by daikk115**
Fix #8

Unmap default CTL+C following: https://github.com/urwid/urwid/blob/master/examples/input_test.py#L84-L89

### CODE REVIEW

File: demoshell/main.py [hunk 1]
```diff
@@ -69,6 +76,13 @@ def on_enter(self, key):
                 self._run_external_command(cmd)
             self.prompt_widget.set_edit_text('')
 
+        elif key == 'ctrl c':
+            if self.last_command and isinstance(
```
Comment (dhellmann) [inline demoshell/main.py]: I don't think we want ctrl-c to terminate the shell. That's not how most shells work, right? I think in the case where the last command has already terminated we can just ignore the ctrl-c or we could ring the bell (or flash the screen).

Comment (daikk115) [reply demoshell/main.py]: Humm,ok, I got it. I think we can ignore and print it out like bellow.
```
daidv@DaiDV ~ $ ^C
daidv@DaiDV ~ $ ^C
daidv@DaiDV ~ $ ^C
```
To exit demoshell, just Ctl+D.

Is this Ok?

Comment (daikk115) [reply demoshell/main.py]: image

I just try and rethink that it's better to ignore without print out anything.

Comment (dhellmann) [reply demoshell/main.py]: Hmm. A message wouldn't be bad, but ignoring the key entirely in the case where there is no active command would be OK as well.

Comment (daikk115) [reply demoshell/main.py]: Ok, I will resolve the conflicts and merge this soon.

**Pull request merged**